iotedgehubdev/cli.py

```diff
@@ -239,7 +248,11 @@ def start(inputs, port, deployment, verbose, host):
             else:
                 input_list = [input_.strip() for input_ in inputs.strip().split(',')]
 
-            edge_manager.start_singlemodule(input_list, port)
+            for env in environment:
```

Comment:
if user didn't specify -e, will it be null for environment? if so, there will be exception

Reply:
it was handled by the click module, and would pass environment as empty tuple, so it would not throw exception